nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -134,14 +148,17 @@ public boolean fromAnnotatedPackage(Symbol.ClassSymbol symbol) {
 
   @Override
   public boolean isExcludedClass(String className) {
-    if (sourceClassesToExclude == null) {
-      return false;
-    }
-    for (String classPrefix : sourceClassesToExclude) {
-      if (className.startsWith(classPrefix)) {
-        return true;
+    if (sourceClassesToExclude != null) {
+      for (String classPrefix : sourceClassesToExclude) {
+        if (className.startsWith(classPrefix)) {
+          return true;
+        }
       }
     }
+    // For optimization purposes, scope of analysis can be limited only to a list of classes.
```

Comment:
I would not have expected the fix serialization configuration to be able to completely disable analysis of classes.  Why can't we just use the other existing mechanisms to exclude classes?

Reply:
I just removed the ```WorkList``` class. With our new optimization technique this is not required anymore and just makes the configuration difficult. 4eb83ddfdcb282861cb9c624a258c5f6e1e0fd2f

Please let me know if you would prefer to have it back.